tutorials/compute-quickstart.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Compute Engine quickstart - Create a to-do app
+description: Use Compute Engine to create a two-tier web application.
+author: jscud
+tags: Compute Engine
+date_published: 2019-04-30
+---
+
+# Compute Engine quickstart
+
+## Build a to-do app with MongoDB
+
+<walkthrough-tutorial-duration duration="15"></walkthrough-tutorial-duration>
+
+<walkthrough-alt>
+Take the interactive version of this tutorial, which runs in the Google Cloud Platform (GCP) Console:
+
+[![Open in GCP Console](https://walkthroughs.googleusercontent.com/tutorial/resources/open-in-console-button.svg)](https://console.cloud.google.com/getting-started?walkthrough_tutorial_id=compute_quickstart)
+
+</walkthrough-alt>
+
+In this quickstart, you'll use Compute Engine to create a two-tier web app.
+The frontend VM runs a Node.js to-do web app, and the backend VM runs MongoDB.
+
+This tutorial guides you through the following:
+
+-   Creating and configuring two VMs
+-   Setting up firewall rules
+-   Using SSH to install packages on your VMs
+
+## Project setup
+
+GCP organizes resources into projects. This allows you to
+collect all of the related resources for a single application in one place.
+
+<walkthrough-project-billing-setup></walkthrough-project-billing-setup>
+
+## Navigate to Compute Engine
+
+Open the [**Navigation menu**][spotlight-console-menu] in the upper-left corner of the console and select 
+**Compute Engine**.
+
+<walkthrough-menu-navigation sectionId="COMPUTE_SECTION"></walkthrough-menu-navigation>
+
+## Create instances
+
+You create two instances for the web app, one to be the backend server and one to be the frontend server.
+
+### Create the backend instance
+
+First, create the backend instance, which runs MongoDB and stores the to-do items:
+
+1.  In the **VM instances** section, click [**Create**][spotlight-create-instance].
+
+1.   Enter a [name][spotlight-instance-name] for this instance.
+
+1.   Choose a region and a [zone][spotlight-instance-zone] for this instance.
+
+1.  In the [**Machine type**][spotlight-machine-type] menu, select **micro**, which specifies a lower-cost
```

Comment:
This said 'f1 micro' before, but that isn't what the UI shows now. I changed it to 'micro' to match the current UI.

Reply:
Sounds great, thank you.